participants/Guerrero/doc/README.md

```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+Ok so I guess you are reading this because you want to use my code. There are some
+functions that do stuff and they are:
```

Comment:
Great start!
You may want to restructure the narration a little bit more, so that it is more descriptive than what it is currently.